### DISCUSSION TITLE
Cant see time picker

### DISCUSSION THREAD

**Discussion by shimonelbaz** [EmilyRosina/gsf-docs#7] [Feedback: Bug] [answered]
when i create a filter from date / date range , i do not see the date picker

**Comment by shimonelbaz** [comment, marked as answer]
nevermind , found it :)

**Reply by EmilyRosina** [reply]
:sweat_smile: :wave: Hiya, ok grand - so you all good for me to mark this as resolved?

**Reply by shimonelbaz** [reply]
Yup! :)
